dtfs-central-api/src/cron-scheduler-jobs/delete-correction-and-correction-request-transient-form-data/index.ts

```diff
@@ -0,0 +1,30 @@
+import { asString, CronSchedulerJob } from '@ukef/dtfs2-common';
+import { FeeRecordCorrectionRequestTransientFormDataRepo } from '../../repositories/fee-record-correction-request-transient-form-data-repo';
+import { FeeRecordCorrectionTransientFormDataRepo } from '../../repositories/fee-record-correction-transient-form-data-repo';
+
+const { RECORD_CORRECTION_TRANSIENT_FORM_DATA_DELETE_SCHEDULE } = process.env;
+
+/**
+ * Deletes record correction and correction request transient form data more than 1 day old.
+ */
+export const deleteCorrectionAndCorrectionRequestTransientFormData = async (): Promise<void> => {
```

Comment:
Perhaps we just call this: `deleteOldCorrectionTransientFormData` or `deleteAllOldCorrectionTransientFormData`. What do you think?